### ISSUE TITLE
test_suite_fft_int32.c fails under armv7

### ISSUE THREAD

**Issue by zhongweiy** [projectNe10/Ne10#100]
@Phillip-Wang This tests fail under ARMv7. It maybe related to recently added complex-non-power-of-2 NEON commit (I'm not sure.).
```
----------test_fft_c2c_1d_float32_conformance start
FFT size 4
test_fft_c2c_1d_float32_conformance Line 120   Should have been false
test_fft_c2c_1d_float32_conformance Line 137   Should have been false
FFT size 8
FFT size 16
FFT size 32
FFT size 64
FFT size 128
FFT size 256
FFT size 512
FFT size 1024
FFT size 2048
FFT size 4096
FFT size 8192
FFT size 16384
FFT size 32768
                           1 run  2 failed                            


ne10/modules/dsp/test/test_suite_fft_int32.c -
----------test_fft_c2c_1d_int32_conformance start
FFT size 4
 ERROR: suffix array guard signature is wrong. 
test_fft_c2c_1d_int32_conformance Line 150   Should have been false
 ERROR: suffix array guard signature is wrong. 
test_fft_c2c_1d_int32_conformance Line 170   Should have been false
 ERROR: suffix array guard signature is wrong. 
test_fft_c2c_1d_int32_conformance Line 190   Should have been false
 ERROR: suffix array guard signature is wrong. 
test_fft_c2c_1d_int32_conformance Line 210   Should have been false
FFT size 8
FFT size 16
FFT size 32
FFT size 64
FFT size 128
FFT size 256
FFT size 512
FFT size 1024
FFT size 2048
FFT size 4096
FFT size 8192
FFT size 16384
FFT size 32768
```

**Comment by Phillip-Wang** [comment]
It is difficult to NEON optimize 4 points FFT (size 4). I suggest we drop support for this case in the future, unless some other libraries are using Ne10 already and they need support for 4 points FFT.

**Comment by Phillip-Wang** [comment]
Fixed in cf33c0d51a445bbe6ad7e21c2af875acee07b838.
